controllers/party_controller.py

```diff
@@ -80,3 +83,23 @@ def get_party_by_email(email):
 
     logger.debug('Successfully retrieved email address', email=email)
     return response.json()
+
+
+def change_respondent_status(respondent_id, status="ACTIVE"):
+    logger.debug('Change respondent account status', respondent_id=respondent_id)
+    request_json = {
+        "status_change": status
+    }
+    url = f'{Config.PARTY_SERVICE}/party-api/v1/respondents/edit-account-status/{respondent_id}'
+    response = requests.put(url, json=request_json, auth=Config.BASIC_AUTH)
+
+    if response.status_code == 404:
+        logger.info('Respondent not found', party_id=respondent_id)
+        return
```

Comment:
Does this need to raise an exception when not found  as below?

Reply:
Maybe just raise_for_status is good enough?